code/__HELPERS/global_lists.dm

```diff
@@ -50,6 +50,13 @@
 
 	init_subtypes(/datum/crafting_recipe, crafting_recipes)
 
+//RPD stuff
+	var/list/pipe_list = list()
+	init_subtypes(/datum/pipes/atmospheric, pipe_list)

```

Comment:
You can just do `var/list/pipe_list = init_subtypes(/datum/pipes/atmospheric)`!

Reply:
Oh, I meant combining it with the line below i.e.

Instead of ```init_subtypes(/datum/pipes/atmospheric, pipe_list)
init_subtypes(/datum/pipes/disposal, pipe_list)```

Do something like `init_subtypes(/datum/pipes/disposal, /datum/pipes/atmospheric, pipe_list)`